tests/headers/convert-cpp-comment-to-rust.hpp

```diff
@@ -0,0 +1,16 @@
+#include <stddef.h>
+#include <stdint.h>
+
+typedef uint32_t mbedtls_mpi_uint;
```

Comment:
instead you can remove them and, let's say, replace `uint32_t` for `unsigned`, and `size_t` for `unsigned long`. they're irrelevant to the test-case anyway, so it seems sensible.